src/main/webapp/app/course/learning-paths/learning-path-graph/learning-path-graph-node.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input } from '@angular/core';
+import { faCheckCircle, faCircle, faPlayCircle, faQuestionCircle } from '@fortawesome/free-solid-svg-icons';
+import { NgxLearningPathNode, NodeType } from 'app/entities/competency/learning-path.model';
+
+@Component({
+    selector: 'jhi-learning-path-graph-node',
+    templateUrl: './learning-path-graph-node.component.html',
+})
+export class LearningPathGraphNodeComponent {
+    @Input() courseId: number;
+    @Input() node: NgxLearningPathNode;
+
+    //icons
```

Comment:
Imho unnecessary

Reply:
What do you mean by unnecessary?
Both this information is passed to child components, e.g. for the different Node details Popovers

Reply:
Sorry for the misunderstanding. You were referring to the comment. I removed it 👍